subsystems/motor/speedmodifiers/AccelerationCap.java

```diff
@@ -29,10 +40,16 @@
 	 * @param hardStopVoltage
 	 *        Voltage to begin decreasing motor speed at.
 	 */
-	public AccelerationCap(PDP pdp, double softStopVoltage, double hardStopVoltage) {
+	public AccelerationCap(PDP pdp, double hardStopVoltage, int... channels) {
 		this.pdp = pdp;
-		this.softStopVoltage = softStopVoltage;
 		this.hardStopVoltage = hardStopVoltage;
+		this.channels = channels;
+		disableCurrent = pdp.getTotalCurrent() > AccelerationCap.REASONABLE_RESTING_CURRENT;
```

Comment:
I am fairly confident that a PDP issue will not appear in the middle of a match, and this is one of the best times that we have a guarantee that all motor controllers are disabled.